### ISSUE TITLE
Deprecated dependency - Ghooks

### ISSUE THREAD

**Issue by eduardomoroni** [infinitered/ignite#660]
[Ghooks](https://github.com/gtramontina/ghooks) is deprecated, now we have [Husky](https://github.com/typicode/husky).

The reasons are [here](https://github.com/gtramontina/ghooks/issues/166).

## Migrating to Husky

To migrate from `ghooks` to `husky`, uninstall `ghooks` and install `husky`:

```sh
npm uninstall ghooks --save-dev
npm install husky --save-dev
```

`ghooks` scripts will be automatically migrated and custom hooks will be preserved.

**Comment by kevinvangelder** [comment]
#584 Fixed this for Ignite 1.0 and we'll probably be updating Ignite 2.0 in the near future.

**Comment by skellock** [comment]
This is now in.
